src/_modules/common_values/locals.tf

```diff
@@ -0,0 +1,76 @@
+locals {
+  # General
+  prefix             = "io"
+  env_short          = "p"
+  location_short     = { westeurope = "weu", italynorth = "itn" }
+  project_itn        = "${local.prefix}-${local.env_short}-${local.location_short.italynorth}"
+  project_weu        = "${local.prefix}-${local.env_short}-${local.location_short.westeurope}"
+  project_weu_legacy = "${local.prefix}-${local.env_short}"
+  core               = data.terraform_remote_state.core.outputs
+
+  #---------#
+  # OUTPUTS #
+  #---------#
+
+  # WHAT: Name and configuration for gates on release, to be changed at each release with the most up-to-date values
+  # HOW: These values ​​will be used for scaling the different resources (function app, app services, etc.)
+  scaling_gate = {
+    name     = "gate1"
+    timezone = "W. Europe Standard Time"
+    start    = "2024-10-23T08:00:00.000Z"
+    end      = "2024-10-23T22:00:00.000Z"
+  }
+
+
+  # WHAT: All resource gorups into itn and weu regions
```

Comment:
It wouldn't be better to move these declarations in the `outputs.tf` file and use the `description` field to add documentation? We could also consider to have multiple output files grouped by some logical boundary

Reply:
Yes, maybe it can be easier to divide the various resources this way!
We could do for now:

- outputs_network.tf
- outputs_general.tf

and then create new ones or modify them based on the outputs we will have